first_year_scraper/excel_formatter.py

```diff
@@ -0,0 +1,137 @@
+from openpyxl import Workbook
+from openpyxl import load_workbook
+import slot
+import json
+
+
+def getkey(a, b):
+    '''returns key for cell[a][b] in excel'''
+    return str(chr(ord('A')+a-1))+str(b)
+
+# copy same values in merged cells
+def sanitise_sheets(valid_sheets, wb):
+    for sheet in valid_sheets:
+        ws = wb.get_sheet_by_name(sheet)
+        ws.delete_cols(1,2)
+        ws.delete_rows(1,2)
+        # merged_ranges = ws.merged_cell_ranges # get all merged ranges in sheet #TAKEN CARE OF BY PARSER
+        # for merged_range in merged_ranges:
+        #     boundary = merged_range.bounds
+        #     original_value = ws[getkey(boundary[0], boundary[1])].value
+        #     ws.unmerge_cells(str(merged_range))
+        #     for i in range(boundary[0], boundary[2]+1):
+        #         for j in range(boundary[1], boundary[3]+1):
+        #             ws[getkey(i,j)].value = original_value
+    # for sheet in valid_sheets:
+    #     ws = wb.get_sheet_by_name(sheet)
+    #     depth = 10
+    #     while(ws['A1'].value != "Period" and depth):
+    #         ws.delete_rows(1)
+    #         depth-=1;
```

Comment:
Hey Akshat, do you think we should remove the commented code?

Ref- https://www.nayuki.io/page/dont-share-commented-out-code

Reply:
Yes! I just kept it so that if the camelot causes an error while parsing in other environments, we can revert to the excel parser from the website I previously mentioned. I'll remove it at the end of the review.